docs/cpp/cmake-linux.md

```diff
@@ -0,0 +1,176 @@
+
+For this tutorial on Ubuntu, we'll use the GCC compiler, GDB to debug, and make to genenerate native build files. These tools are not installed by default on Ubuntu, so you need to install them. Fortunately, that's easy.
+
+### Check if GCC is installed
+
+ To see if GCC is already installed on your system, open a Terminal window and enter the following command:
+
+```bash
+gcc -v
+```
+
+If GCC isn't installed, run the following command from the terminal window to update the Ubuntu package lists. An out-of-date Linux distribution can interfere with getting the latest packages.
+
+```bash
+sudo apt-get update
+```
+
+Next, install the GNU compiler, make, and the GDB debugger with this command:
+
+```bash
+sudo apt-get install build-essential gdb
+```
+
+## Create a CMake quickstart
+
+If you have an existing CMake project that already has a CMakeLists.txt file in the root directory, you can skip to [Select a kit](#select-a-kit) to configure your existing project.
+
+Otherwise, create a folder for a new project. From the terminal window, create an empty folder called `cmakeQuickStart`, navigate into it, and open VS Code in that folder by entering the following commands:
+
+```cmd
+mkdir cmakeQuickStart
+cd cmakeQuickStart
+code .
+```
+
+The `code .` command opens VS Code in the current working folder, which becomes your "workspace".
+
+### Create a CMake hello world project
+
+The CMake Tools extension can create the files for a basic CMake project for you. Open the Command Palette (`kb(workbench.action.showCommands)`) and run the **CMake: Quick Start** command:
+
+![Create CMake quickstart](images/cpp/cmake-quickstart-command-palette.png)
+
+Enter a project name. This will be written to `CMakeLists.txt` and a few initial source files.
+
+Next, select **Executable** as the project type to create a basic source file (`main.cpp`) that includes a basic `main()` function.
+
+![Choose project type](images/cpp/cmake-choose-type.png)
+
+**Note:** If you had wanted to create a basic source and header file, you would have selected **Library** instead. But for this tutorial, **Executable** will do. If you are prompted to configure IntelliSense for the folder, you can either ignore the prompt or select **Allow**.
+
+This creates a hello world CMake project containing `main.cpp`, `CMakeLists.txt` (which tells the CMake tools how to build your project), and a folder named **build** for your build files:
+
+![Project contents](images/cpp/cmake-project-contents.png)
+
+### Select a kit
+
+Before you can use the CMake tools extension to build a project, you need to configure it to know about the compilers on your system. Do that by scanning for what CMake calls 'kits'. A kit represents a toolchain, which is the compilers, linkers, and other tools used to build your project. To scan for kits:
```

Comment:
CMake Tools ("Tools" with a capital "T").
`CMake calls 'kits'` --> `CMake Tools calls 'kits'`  (CMake itself doesn't use this terminology)

Reply:
Fixed. I think I have them all.

Reply:
Fixed. I think I have them all.

Reply:
removed "cmake tools calls" and just say "scanning for 'kits'", since this is what the extension calls them.